src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -1,13 +1,11 @@
-import environment from 'platform/utilities/environment';
```

Comment:
the environment helper only works in the browser so it can't be used for default value setting in a node app. I'd already set up the constructor to take a hostname value so I opted to just hardcode the default to `localhost` for now.